packages/mux-player-react/src/news/index.tsx

```diff
@@ -60,11 +73,11 @@ const MuxNewsPlayer = ({ videoList, ...props }: PlaylistProps) => {
       {...props}
       ref={mediaElRef}
       key={`player-${playerKey}`}
-      playbackId={videoList[currentIndex].playbackId}
-      adTagUrl={videoList[currentIndex].adTagUrl}
+      playbackId={currentAdTagUrlString ? videoList[currentIndex].playbackId : undefined}
```

Comment:
is this condition still needed? 
I merged a fix so the adsLoader is available on connectedCallback 
[`1c289f6` (#1128)](https://github.com/muxinc/elements/pull/1128/commits/1c289f65cb1a72452aa69efa6cf0392e55cf6836)

Reply:
Yes, still needed. This is the light lift way of blocking loading the new `playbackId` until the async scenarios for `adTagUrl` are resolved.